tests/unit/test_public_shims.py

```
@@ -650,6 +668,36 @@ def test_deprecated_top_level_studio_content_type_import_warns_and_caches(
+    monkeypatch: pytest.MonkeyPatch,
+) -> None:
+    """from notebooklm import DEFAULT_STORAGE_PATH keeps the deprecated shim contract."""
+    import notebooklm
+    from notebooklm.paths import get_storage_path
+
+    monkeypatch.delitem(notebooklm.__dict__, "DEFAULT_STORAGE_PATH", raising=False)
+
+    with warnings.catch_warnings(record=True) as caught:
+        warnings.simplefilter("always", DeprecationWarning)
+        namespace: dict[str, object] = {}
+        exec("from notebooklm import DEFAULT_STORAGE_PATH", namespace)
```

Ruff (0.15.12)

[error] Use of `exec` detected

(S102)